packages/@lwc/perf-benchmarks/src/__benchmarks__/engine-dom/benchmark-table-component/tablecmp-create-1k.benchmark.js

```diff
@@ -11,7 +11,7 @@ import Table from '@lwc/perf-benchmarks-components/dist/dom/benchmark/tableCompo
 import Store from '@lwc/perf-benchmarks-components/dist/dom/benchmark/store/store.js';
 import { insertComponent, destroyComponent } from '../../../utils/utils.js';
 
-benchmark(`benchmark-table-component/create/1k`, () => {
+benchmark(`dom/table-component/create/1k`, () => {
```

Comment:
Not sure of the value of this benchmark since we already have `dom/table-component/create/10k`. But we can delete it later if we want.